apps/web/src/app/[lang]/profile/page.tsx

```diff
@@ -81,7 +81,11 @@ export default function Profile() {
           <ProposalsSection basePath="" useDocuments={useSpaceDocuments} />
         </TabsContent>
         <TabsContent value="discussions">
-          <DiscussionsSection basePath="" useDocuments={useSpaceDocuments} />
+          <DocumentSection
+            basePath=""
```

Comment:
Base path missing. Please create a ticket and describe what's necessary to implement.

Reply:
@plitzenberger Yes, they are empty now because to build basePath we need two variables lang and id. We have lang, but we cannot get the space id, since the profile page is hierarchically located above the space pages.

Reply:
Added comment and ticket